### ISSUE TITLE
SCC is not allowing non-random UID 

### ISSUE THREAD

**Issue by thesteve0** [minishift/minishift#586]
For minishift I though were supposed to allow root and non-random UID for containers. When I try that I get:

```
PS C:\Users\steve\OneDrive\Documents\GitHub\crunchy-containers\examples\openshift\workshop> oc new-app -p CCP_IMAGE_TAG=
centos7-9.5-1.2.6 -f master.json
--> Deploying template "myproject/master" for "master.json" to project myproject

     master
     ---------
     PostgreSQL Replication Example


     * With parameters:
        * CCP_IMAGE_TAG=centos7-9.5-1.2.6

--> Creating resources ...
    service "master" created
    error: pods "master" is forbidden: unable to validate against any security context constraint: [fsGroup: Invalid val
ue: [26]: 26 is not an allowed group]
--> Failed
```

Please change the SCC to allow non-random UIDs to allow developer to be productive

**Comment by thesteve0** [comment]
in the meantime, fixed it from my host machine with this

 oc adm policy add-scc-to-group anyuid system:authenticated --as system:admin

**Comment by gbraad** [comment]
As far as I remember, we did not allow non-root containers, as this would introduce possible issues when people would move from their local development to eg. OpenShift Online

**Comment by thesteve0** [comment]
I understand that logic but if you go down that path expect a lot of issues
for people just "trying it out". I think it is easier to make life easier
for them to get started. Also warn them, like we used to do on the all in
one VM, that we had relaxed permissions and that this would not work on a
typical enterprise cluster.

**Comment by LalatenduMohanty** [comment]
@thesteve0 This is something we should be able to handle once we have fix for #177 (as per my understanding).  The idea is to provide users different environments i.e. an easy environment for them to try out things as you suggested, another environment similar to openshift online etc. So that users will be able to emulate the environment they want.  cc @hferentschik   

CC @hferentschik

**Comment by hferentschik** [comment]
@thesteve0, this indeed something which I am addressing as part of #177. In fact I am using this particular use case as my main use case/example for the implementation.

**Comment by Preeticp** [comment]
This issue was resolved as part of #177, closing this issue.
